src/components/EmojiPicker/EmojiPicker.js

```diff
@@ -56,6 +56,11 @@ class EmojiPicker extends React.Component {
      * @param {String} emoji
      */
     selectEmoji(emoji) {
+        // Prevent fast click / multiple emoji selection
+        if (!this.emojiPopoverAnchor) {
```

Comment:
Can you explain where/how `this.emojiPopoverAnchor` is set to `null` here?

Reply:
@jasperhuangg Is you mean adding the explanation to the comment? If so, I also agree that it will give awareness about the condition there.

Reply:
@s77rt please don't resolve comments without responding

I wanted you to update the comment to include  a more clear explanation of where/how `this.emojiPopoverAnchor` is set to null